src/main/java/frc/robot/subsystems/intake/IntakeIOSparkMax.java

```diff
@@ -59,4 +67,15 @@ public void setIntakeVoltage(double volts) {
     public void setBeltVoltage(double volts) {
         belt.setControl(new VoltageOut(volts));
     }
+
+    @Override
+    public boolean checkIfStopIntake() {
```

Comment:
We're going to add a ToF sensor here so we can use that instead of current sensing and a timer.